proto/gk/v1/gatekeeper.proto

```diff
@@ -241,6 +251,15 @@ message RemoveCollectionMemberRequest {
 
 message RemoveCollectionMemberResponse {}
 
+// Events
+message Event {
+	string                    name      = 1;
+	google.protobuf.Any       payload   = 3;
+	google.protobuf.Timestamp timestamp = 2;
+}
+
+message ConsumeEventResponse {}
```

Comment:
why is an empty response needed? can't use the pb empty?

Reply:
We could've just used `google.protobuf.Empty` here but if we ever want to update this endpoint to return some data it'll be a breaking change to convert from `google.protobuf.Empty` to something else. So this is mainly to keep backwards compatibility.

Reply:
right. I guess if the endpoint would be used for testing, might make sense to return something at some point. I wasn't sure why we'd return anything to Pub/Sub ever